### DISCUSSION TITLE
Istio 1.24.1 Internal AWS Load Balancer issue.

### DISCUSSION THREAD

**Discussion by mp365** [istio/istio#54261] [Q&A]
Hello, 

I'm installing Istio for the first time and am having trouble getting an ingress Gateway working with an Internal AWS NLB. 

The Kubernetes cluster is v1.30 and I'm using Istio 1.24.1. 

I have tried several installs using Helm and istioctl. All combinations I have tried so far seem to result in the same scenario. I can access my basic application deployment when I curl the LB URL. However the issue is it can take >3 minutes to respond. 

My current install was performed using the following commands: 

`$ istioctl x precheck` 
`✔ No issues found when checking the cluster. Istio is safe to install or upgrade!`

`istioctl --set profile=minimal`
`This will install the Istio 1.24.1 profile "minimal" into the cluster. Proceed? (y/N) y`
`✔ Istio core installed ⛵️`
`✔ Istiod installed 🧠`
`✔ Installation complete`

I then used the basic Gateway[ ingress.yaml](https://istio.io/latest/docs/setup/additional-setup/gateway/) from the Istio documentation to install a Gateway in the istio-ingress namespace. 

This successfully created a NLB and a Target Group listening on port 80. 

I created a basic Nginx Deployment with a Service, Gateway and Virtual Service resources. the Nginx deployment simply serves the standard 'Welcome' index.html. 

One thing to note in this configuration is that only one of the Nodes is showing as Healthy in the in the load balancer target group.

I'd be grateful if anyone could help troubleshoot  this scenario or offer any common suggestions as to what might be misconfigured?

Why would one node show as healthy and the other not (the under lying ec2 intstances share the same security group which has been updated with the node port the Istio gateway service is listening on). 
Why would the page take so long to respond? 

Thanks in advance
Mark

**Comment by BeyondEvil** [comment]
I'm new to Istio myself, but if I'm reading the documentation correctly, when you use the "minimal" profile, the `ingressgateway` isn't installed as it's expected you will use the Istio implementation of Kubernetes Gateway API. So you would use `HTTPRoute` instead of `VirtualService`.
